tests/test_format.py

```diff
@@ -225,26 +225,27 @@ def test_format_comparison_bars_to_js_exceptions(self):
             self.comparison_coords_headers, self.comparison_coords_data, 11)
 
     def test_format_emperor_html_footer_string(self):
+        self.maxDiff=5000
         """Test correct formatting of the footer string"""
         # footer for a jackknifed pcoa plot without biplots
         out_string = format_emperor_html_footer_string(False, True)
-        self.assertEqual(out_string, EXPECTED_FOOTER_A)
+        self.assertItemsEqual(out_string.split('/'), EXPECTED_FOOTER_A.split('/'))
```

Comment:
Its just a way to make debugging easier.  Before, an entire HTML page would be dumped on the page.  Using this will allow only portions of differing HTML to be displayed.

It isn't a big though - I can remove it.